src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderRequest.cs

```diff
@@ -27,10 +28,11 @@ public class PaymentOrderRequest
                                    Language language,
                                    bool generateRecurrenceToken,
                                    IUrls urls,
-                                   IPayeeInfo payeeInfo)
+                                   IPayeeInfo payeeInfo,
+                                   bool useCheckoutV3 = false)

```

Comment:
I feel that `bool useCheckoutV3` does not scale very well. What should we do when we launch Checkout v4? I would like to explore other designs for this, perhaps by adding a `PaymentOrderRequestBuilder` which can reduce the number of arguments in `PaymentOrderRequest` as well as make it immutable.

Reply:
Can you please just resolve this and complete this PR? Swedbank does not want us to use Checkout v2 any more.